### PR TITLE
FLUT-6089 - [Bug] UG improvements

### PR DESCRIPTION
**Pull request by PavithraNandhiniRamachandran**
## Description ##

UG improvements

### CODE REVIEW

File: Flutter/cartesian-charts/axis-types.md [hunk 1]
```diff
@@ -65,17 +78,40 @@ By using the [`isInversed`](https://pub.dev/documentation/syncfusion_flutter_cha
...
+        final List<ChartData> chartData = <ChartData>[
+            ChartData(2010, 10.53),
+            ChartData(2011, 9.5),
+            ChartData(2012, 10),
+            ChartData(2013, 9.4),
+            ChartData(2014, 5.8),
+            ChartData(2015, 4.9),
+            ChartData(2016, 4.5),
+            ChartData(2017, 3.6),
+            ChartData(2018, 3.43),
```
Comment (dharanidharandharmasivam) [inline Flutter/cartesian-charts/axis-types.md:90]: Is the data is fine, check with the below image
image

File: Flutter/cartesian-charts/axis-types.md [hunk 2]
```diff
@@ -158,7 +213,13 @@ When the value of [`rangePadding`](https://pub.dev/documentation/syncfusion_flut
                         primaryYAxis: NumericAxis(
                             // Additional range padding is applied to y axis
                             rangePadding: ChartRangePadding.additional
-                        )  
+                        ),
+                        series: <ChartSeries<ChartData, int>>[
```
Comment (dharanidharandharmasivam) [inline Flutter/cartesian-charts/axis-types.md:217]: why do we need to add the series without data?

File: Flutter/cartesian-charts/chart-types/area-chart.md [hunk 3]
```diff
@@ -82,10 +93,6 @@ The borders of the area chart can be customized using the [`borderDrawMode`](htt
 
 {% endhighlight %}
 
-![Area border](cartesian-chart-types-images/area_border.jpg)
```
Comment (dharanidharandharmasivam) [inline Flutter/cartesian-charts/chart-types/area-chart.md:85]: why we need to remove the image

File: Flutter/cartesian-charts/chart-types/spline-area-chart.md [hunk 4]
```diff
@@ -116,13 +150,15 @@ The following code sample demonstrates how to set the `splineType` value to `car
...
+#### See Also
 
-* [Color palette](./series-customization#color-palette) 
-* [Color mapping](./series-customization#color-mapping-for-data-points)
-* [Animation](./series-customization#animation)
-* [Gradient](./series-customization#gradient-fill)
-* [Empty points](./series-customization#empty-points)  
-* [Sorting](./series-customization##sorting) 
\ No newline at end of file
+* [color palette](https://help.syncfusion.com/flutter/cartesian-charts/series-customization#color-palette)
```
Comment (dharanidharandharmasivam) [inline Flutter/cartesian-charts/chart-types/spline-area-chart.md:159]: I didn't asked to change the link like this, the existing itself fine, why you have changed like this?

**Review by dharanidharandharmasivam — CHANGES_REQUESTED** [review]
Hi @PavithraNandhiniRamachandran ,

I have added comments, check for the same in all your changes. And in the inversed numeric axis image, you have added new image, but its not invested

image

Thanks,
Dharani,

File: Flutter/cartesian-charts/chart-types/spline-area-chart.md (hunk 4 above)
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/chart-types/spline-area-chart.md:159]: Hi @dharanidharandharmasivam ,

existing link is not open properly, only the above type only opens, so I have changed.

regards,
@PavithraNandhiniRamachandran

File: Flutter/cartesian-charts/axis-types.md (hunk 2 above)
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/axis-types.md:217]: In `### Customizing range` datasource and below data's are same, so I didn't add.

File: Flutter/cartesian-charts/chart-types/area-chart.md (hunk 3 above)
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/chart-types/area-chart.md:85]: have revert this change

File: Flutter/cartesian-charts/axis-types.md (hunk 1 above)
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/axis-types.md:90]: have added the proper image

File: Flutter/cartesian-charts/axis-types.md [hunk 5]
```diff
@@ -27,30 +27,43 @@ Numeric axis uses numerical scale and displays numbers as labels. By default, [`
 
     @override
     Widget build(BuildContext context) {
+        final List<ChartData> chartData = [
+            ChartData(1, 235, 240),
+            ChartData(2, 242, 250),
+            ChartData(3, 320, 280),
+            ChartData(4, 310, 305),
```
[the remark this answers is not in the document]
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/axis-types.md]: have added the proper data

File: Flutter/cartesian-charts/chart-types/spline-range-area-chart.md [hunk 6]
```diff
@@ -68,62 +80,62 @@ The following code sample demonstrates how to set the [`splineType`](https://pub
...
-## See Also
+#### See Also
 
-* [Color palette](./series-customization#color-palette) 
-* [Color mapping](./series-customization#color-mapping-for-data-points)
-* [Animation](./series-customization#animation)
-* [Gradient](./series-customization#gradient-fill)
-* [Empty points](./series-customization#empty-points)  
-* [Sorting](./series-customization##sorting) 
+* [color palette](https://help.syncfusion.com/flutter/cartesian-charts/series-customization#color-palette)
```
Comment (dharanidharandharmasivam) [inline Flutter/cartesian-charts/chart-types/spline-range-area-chart.md:136]: why do we need to give this full link, I already sked to give proper link

File: Flutter/cartesian-charts/getting-started.md [hunk 7]
```diff
@@ -58,7 +58,7 @@ Import the following package in your Dart code.
 
 Once the package has been imported, initialize the chart as a child of any widget. SfCartesianChart is used to render all kinds of charts which need to be plotted in Cartesian coordinates. Here, as we are plotting line chart, initialize SfCartesianChart widget as a child of Container widget.
 
-{% highlight dart %} 
+{% highlight dart hl_lines="7" %}
```
Comment (dharanidharandharmasivam) [inline Flutter/cartesian-charts/getting-started.md:61]: revert this change

File: Flutter/cartesian-charts/selection.md [hunk 8]
```diff
@@ -116,19 +133,53 @@ The selection features allows you to select segments in following modes using [`
 
 {% highlight dart %} 
 
+    late SelectionBehavior _selectionBehavior;
+    
+    @override
+    void initState(){
+     _selectionBehavior = SelectionBehavior(
+            enable: true);
```
Comment (dharanidharandharmasivam) [inline Flutter/cartesian-charts/selection.md:141]: align this code snippet

File: Flutter/cartesian-charts/selection.md [hunk 9]
```diff
@@ -139,15 +190,38 @@ The selection features allows you to select segments in following modes using [`
 Multiple selection can be enabled using the [`enableMultiSelection`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/SfCartesianChart/enableMultiSelection.html) property of chart.
 
 {% highlight dart %} 
+    late SelectionBehavior _selectionBehavior;
+    
+    @override
+    void initState(){
+     _selectionBehavior = SelectionBehavior(
+            enable: true);
```
Comment (dharanidharandharmasivam) [inline Flutter/cartesian-charts/selection.md:198]: align the code snippet in all places

File: Flutter/cartesian-charts/tooltip.md [hunk 10]
```diff
@@ -119,6 +119,7 @@ By default, x and y value will be displayed in the tooltip, and it can be custom
 * Y value - `point.y`
 * Bubble size - `point.size`
 * Name of the series - `series.name`
+* Stores the cumulative value - `point.cumualtive `
```
Comment (dharanidharandharmasivam) [inline Flutter/cartesian-charts/tooltip.md:122]: Stores the cumulative value for stacked series

File: Flutter/circular-charts/getting-started.md [hunk 11]
```diff
@@ -58,7 +58,7 @@ Import the following package in your Dart code.
 
 Once the package has been imported, initialize the chart as a child of any widget. SfCircularChart can be used to render pie, doughnut and radial bar charts. Here, as we are rendering pie chart, initialize SfCircularChart widget as a child of Container widget.
 
-{% highlight dart %} 
+{% highlight dart hl_lines="7" %}
```
Comment (dharanidharandharmasivam) [inline Flutter/circular-charts/getting-started.md:61]: revert this

File: Flutter/funnel-chart/getting-started.md [hunk 12]
```diff
@@ -58,7 +58,7 @@ Import the following package in your Dart code.
 
 Once the package has been imported, initialize the chart as a child of any widget. SfFunnelChart can be used to render Funnel charts. Here, as we are rendering Funnel chart, initialize SfFunnelChart widget as a child of Container widget.
 
-{% highlight dart %} 
+{% highlight dart hl_lines="7" %}
```
Comment (dharanidharandharmasivam) [inline Flutter/funnel-chart/getting-started.md:61]: revert this

File: Flutter/pyramid-chart/getting-started.md [hunk 13]
```diff
@@ -58,7 +58,7 @@ Import the following package in your Dart code.
 
 Once the package has been imported, initialize the chart as a child of any widget. SfPyramidChart can be used to render pyramid charts. Here, as we are rendering pyramid chart, initialize SfPyramidChart widget as a child of Container widget.
 
-{% highlight dart %} 
+{% highlight dart hl_lines="7" %}
```
Comment (dharanidharandharmasivam) [inline Flutter/pyramid-chart/getting-started.md:61]: revert this

**Review by dharanidharandharmasivam — APPROVED** [review]
Hi @JayavigneshwaranG ,

Kindly merge the PR.

Thanks,
Dharani.

File: Flutter/cartesian-charts/axis-types.md [hunk 14]
```diff
@@ -483,17 +599,24 @@ The date-time axis uses date-time scale and displays date-time values as axis la
...
                         primaryXAxis: DateTimeAxis(),
-                         series: <ChartSeries<SalesData, DateTime>>[
+                         series: <ChartSeries<ChartData, DateTime>>[
                             // Renders line chart
-                            LineSeries<SalesData, DateTime>(
+                            LineSeries<ChartData, DateTime>(
                                 dataSource: chartData,
-                                xValueMapper: (SalesData sales, _) => sales.year,
-                                yValueMapper: (SalesData sales, _) => sales.sales
+                                xValueMapper: (ChartData sales, _) => sales.year,
```
[the remark this answers is not in the document]
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/axis-types.md:618]: have renamed as data

File: Flutter/cartesian-charts/axis-types.md [hunk 15]
```diff
@@ -907,9 +1083,9 @@ Logarithmic axis uses logarithmic scale and displays numbers as axis labels.
         );
     }
 
-    class SalesData{
-        SalesData(this.year, this.sales);
-        final double year;
+    class ChartData{
+        ChartData(this.number, this.sales);
```
[the remark this answers is not in the document]
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/axis-types.md:1087]: have resolved the above concern

File: Flutter/cartesian-charts/getting-started.md (hunk 7 above)
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/getting-started.md:61]: the above change was revert

File: Flutter/cartesian-charts/chart-types/ohlc-chart.md [hunk 16]
```diff
@@ -33,32 +33,7 @@ To render a OpenHighLowClose chart, create an instance of [` hiloOpenCloseSeries
...
-
-                            )
-                        ]
-                    )
-                )   
-            )
-        );
-    }
-
+{% include_relative code-snippet/ohlc.cs %}
```
[the remark this answers is not in the document]
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/chart-types/ohlc-chart.md:36]: removed the above changes

File: Flutter/cartesian-charts/selection.md (hunk 8 above)
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/selection.md:141]: aligned properly

File: Flutter/cartesian-charts/selection.md (hunk 9 above)
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/selection.md:198]: aligned properly

File: Flutter/cartesian-charts/tooltip.md (hunk 10 above)
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/tooltip.md:122]: resolved the above concerns

File: Flutter/pyramid-chart/getting-started.md (hunk 13 above)
Comment (PavithraNandhiniRamachandran) [reply Flutter/pyramid-chart/getting-started.md:61]: revert the above change

File: Flutter/funnel-chart/getting-started.md (hunk 12 above)
Comment (PavithraNandhiniRamachandran) [reply Flutter/funnel-chart/getting-started.md:61]: revert the above change

File: Flutter/circular-charts/getting-started.md (hunk 11 above)
Comment (PavithraNandhiniRamachandran) [reply Flutter/circular-charts/getting-started.md:61]: revert the above change

File: Flutter/cartesian-charts/chart-types/spline-range-area-chart.md (hunk 6 above)
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/chart-types/spline-range-area-chart.md:136]: I have changed the link to below, also the error was occur
image
